pegasus/helpers/section_api_helpers.rb

```diff
@@ -485,6 +485,7 @@ def script
   def to_owner_hash
     to_member_hash.merge(
       script: script,
+      course_id: @row[:course_id],
```

Comment:
Right. There should probably be course caches in both dashboard (my link) and pegasus (your link).